### ISSUE TITLE
GNU Lesser General Public License v3.0 or later

### ISSUE THREAD

**Issue by kashike** [KyoriPowered/indra#31]
https://spdx.org/licenses/LGPL-3.0-or-later.html

**Comment by zml2008** [comment]
```gradle
  license {
    name("GNU Lesser General Public License v3.0 or later")
    url("https://www.gnu.org/licenses/lgpl-3.0.html")
    spdx("LGPL-3.0-or-later")
  }
```

**Comment by zml2008** [comment]
Fixed by #38
